srcpkgs/consolation/template

```diff
@@ -0,0 +1,17 @@
+# Template file for ''consolation''
+pkgname='consolation'
+version='0.0.8'
+revision='1'
+build_style='gnu-configure'
+hostmakedepends='pkg-config>=0.9 libinput-devel>=1.15'
+depends='libinput>=1.15'
```

Comment:
`depends` can be removed, it links libinput and xbps-src automatically adds the dependency.